notebooks/Hello World.ipynb

```diff
@@ -17,6 +17,62 @@
    "source": [
     "print(\"Hello World!\")"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae684151",
+   "metadata": {},
+   "source": [
+    "Super important information here."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "ae09e18d",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import vertexai\n",
+    "from vertexai.generative_models import (\n",
+    "    GenerationConfig,\n",
+    "    GenerativeModel,\n",
+    "    HarmBlockThreshold,\n",
+    "    HarmCategory,\n",
+    "    Part,\n",
+    ")\n",
+    "\n",
+    "vertexai.init()\n",
+    "\n",
+    "MODEL_ID = \"gemini-1.5-flash-preview-0514\"  # @param {type:\"string\"}\n",
+    "\n",
+    "model = GenerativeModel(MODEL_ID)\n",
+    "\n",
+    "# Load a example model with system instructions\n",
+    "example_model = GenerativeModel(\n",
+    "    MODEL_ID,\n",
+    "    system_instruction=[\n",
+    "        \"You are a cheerful professor.\",\n",
+    "    ],\n",
+    ")\n",
+    "\n",
+    "prompt = \"\"\"\n",
+    "  User input: Why is the sky blue?\n",
+    "  Answer:\n",
+    "\"\"\"\n",
+    "\n",
+    "\n",
+    "# Set contents to send to the model\n",
+    "contents = [prompt]\n",
+    "\n",
```

Comment:
This code should be commented.  Explain what this code block is doing, why it's here, and what it's expected to do.